plugins/webapi/tools/plugin.go

```diff
@@ -13,6 +13,21 @@ import (
 // PluginName is the name of the web API tools endpoint plugin.
 const PluginName = "WebAPI tools Endpoint"
 
+const (
+	routeDiagnostics                           = "tools/diagnostic"
+	RouteDiagnosticMessages                    = routeDiagnostics + "/messages"
+	RouteDiagnosticsFirstWeakMessageReferences = RouteDiagnosticMessages + "/firstweakreferences"
+	RouteDiagnosticsMessageRank                = RouteDiagnosticMessages + "/rank/:rank"
+	RouteDiagnosticsUtxoDag                    = routeDiagnostics + "/utxodag"
+	RouteDiagnosticsBranches                   = routeDiagnostics + "/branches"
+	RouteDiagnosticsLazyBookedBranches         = RouteDiagnosticsBranches + "/lazybooked"
+	RouteDiagnosticsInvalidBranches            = RouteDiagnosticsBranches + "/invalid"
+	RouteDiagnosticsTips                       = routeDiagnostics + "/tips"
+	RouteDiagnosticsStrongTips                 = RouteDiagnosticsTips + "/strong"
+	RouteDiagnosticsWeakTips                   = RouteDiagnosticsTips + "/weak"
+	RouteDiagnosticsDRNG                       = routeDiagnostics + "/drng"
```

Comment:
Please add a comment to each of those to make the go linter happy or, in alternative, make them unexported